spec/unit/show-hide-content.spec.js

```diff
@@ -225,6 +273,10 @@ describe('show-hide-content', function () {
         }))
         expect(events && events.click).toContain(jasmine.objectContaining({
           namespace: 'ShowHideContent',
+          selector: 'input[type="radio"][name="test.single"]'
+        }))
+        expect(events && events.click).toContain(jasmine.objectContaining({
```

Comment:
That's fair enough. Seems like all the other tests do that too. Would probably better to refactor this separately you're right.